roles/monit/templates/monit.conf.j2

```diff
@@ -1,16 +1,15 @@
 {{ ansible_managed | comment }}
-check process {{ apache_service_name }}
-  with pidfile /var/run/{{ apache_service_name }}/{{ apache_service_name }}.pid
+check process {{ apache_service_name }} with pidfile /var/run/{{ apache_service_name }}/{{ apache_service_name }}.pid
   start program = "/usr/sbin/service {{ apache_service_name }} start"
   stop program  = "/usr/sbin/service {{ apache_service_name }} stop"
-  if failed port 80 protocol http
```

Comment:
@kayiwa why did you remove the timeout information here.  I think that might still be needed to notice apache is having an issue.